tests/migration_test.go

```diff
@@ -520,35 +512,17 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			metrics, err = parseMetricsToMap(lines)
 			Expect(err).ToNot(HaveOccurred())
 			return metrics
-		}, 100*time.Second, 1*time.Second).ShouldNot(BeEmpty())
-
-		Expect(len(metrics)).To(BeNumerically(">=", 1.0))
+		}, 100*time.Second, 1*time.Second).ShouldNot(BeEmpty(), "no metrics with kubevirt_migrate_vmi prefix are found")
 		Expect(metrics).To(HaveLen(len(lines)))
 
 		By("Checking the collected metrics")
 		keys := getKeysFromMetrics(metrics)
 		for _, key := range keys {
 			value := metrics[key]
-			fmt.Fprintf(GinkgoWriter, "metric value was %f\n", value)
-			Expect(value).To(BeNumerically(">=", float64(0.0)))
+			Expect(value).ToNot(BeZero(), "metric value is not expected to be zero")
```

Comment:
Is this correct?

Reply:
Well, in theory these values could be negative (since it's a `float` type).
In practice, these metrics should never be positive as they describe things like how many VMs are pending, how much data has been transferred, etc. The purpose of this check if to ensure the metrics are non-zero (or IOW, had been initialized and show actual data).

If you think it's better, I'll revert this.